stl/inc/list

```diff
@@ -143,7 +143,7 @@ public:
     _NODISCARD reference operator*() const noexcept {
 #if _ITERATOR_DEBUG_LEVEL == 2
         const auto _Mycont = static_cast<const _Mylist*>(this->_Getcont());
-        _STL_ASSERT(_Mycont, "cannot dereference value-initialized list iterator");
+        _STL_VERIFY(_Mycont, "cannot dereference value-initialized list iterator");

```

Comment:
suggest `_Mycont != nullptr` instead of just `_Mycont`

Reply:
Thank you, but our stylistic conventions both allow and often prefer directly testing pointers for non-nullness. (It's shorter, avoids the negation in `!=`, and reads naturally; an expression like `if (ptr && ptr->whatever)` reads as "if we have `ptr` and its `whatever` is true...")

`_STL_VERIFY` is well-known to us as testing its condition, so I view this context as equivalent to an `if` statement.

(But conversely, our stylistic conventions do not allow directly testing integers for non-zeroness, except in special circumstances.)